mithril-client/src/services/mithril_stake_distribution.rs

```diff
@@ -153,24 +153,22 @@ impl MithrilStakeDistributionService for AppMithrilStakeDistributionService {
         genesis_verification_key: &str,
     ) -> StdResult<PathBuf> {
         // 1 - retrieve stake distribution
```

Comment:
Maybe you could also remove all these numbered comments that don't bring a lot here